deploy/olm-catalog/kogito-operator/kogito-operator.package.yaml

```diff
@@ -1,7 +1,7 @@
 channels:
-- currentCSV: kogito-operator.v0.7.0
```

Comment:
Should the alpha version not stay in 0.7.0 ?

Reply:
It's set to the next version

Reply:
alpha should not stay as 0.7.0 as long as 0.8.0 has not been released or ?

Reply:
Won't be a problem since this file is only used for the PR to OLM.